committeeoversightapp/utils.py

```diff
@@ -53,3 +56,73 @@ def save_document(url, note, event):
             new_archived_document_link.save()
 
         return new_document
+
+def get_document_context(context):
+    document_types = {'transcript':'transcript', 'opening_statement_chair':'chair opening statement', 'opening_statement_rm':'ranking member opening statement'}
```

Comment:
Can you format this, so that each key-value pair is on a line? 

```
{'transcript':'transcript',
 'opening_statement_chair':'chair opening statement',
....
```